### PR TITLE
Stop generating challenges is there are no validators

### LINKED ISSUE #1060: stop to generate challenge if there is no any valid validator
```
github.com/0chain/blobber/code/go/0chain.net/blobbercore/challenge.loadValidationTickets
	/Users/geax/code/geax/github.com/0chain/blobber/code/go/0chain.net/blobbercore/challenge/challenge.go:152
github.com/0chain/blobber/code/go/0chain.net/blobbercore/challenge.processAccepted.func2
	/Users/geax/code/geax/github.com/0chain/blobber/code/go/0chain.net/blobbercore/challenge/challenge.go:123
2022-03-14T21:07:52.210+0800	ERROR	challenge/challenge.go:125	[challenge]validate: 	{"challenge_id": "f565c2a58d1092b79841cd199edc173d436e08f9b1bd8dccc41c826a2ac7fea2", "error": "no_validators: No validators assigned to the challenge"}
github.com/0chain/blobber/code/go/0chain.net/blobbercore/challenge.processAccepted.func2
	/Users/geax/code/geax/github.com/0chain/blobber/code/go/0chain.net/blobbercore/challenge/challenge.go:125
```

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
## Changes
Added check on size of validators list to stop generating challenges if there are no validators.

## Fixes
Resolves #1060 

## Tests 
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | enhance/challenge-generation-no-validator  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: .github/workflows/system_tests.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ on:
         required: true
 jobs:
   system-tests:
-    uses: 0chain/actions/.github/workflows/manual_system_tests.yml@master
+    uses: 0chain/actions/.github/workflows/manual_system_tests.yml@check-without-validators
```
Comment (peterlimg) [inline .github/workflows/system_tests.yml]: Do not change this, we can select this branch by running system test manually

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Comment by ma2b0043** [comment]
@Kishan-Dhakan @peterlimg when can we expect to close this PR?

**Review by peterlimg — APPROVED** [review]
lgtm

**Comment by Kishan-Dhakan** [comment]
> @Kishan-Dhakan @peterlimg when can we expect to close this PR?

Was awaiting some reviews, done @ma2b0043 👍

**Pull request merged**